applications/GeoMechanicsApplication/tests/Simple_Dike_Gravity_Loading/line_body_all_stage_new_units_kPa/ProjectParameters_stage2.json

```
@@ -12,8 +12,7 @@
         "model_part_name":                    "PorousDomain",
         "domain_size":                        2,
         "model_import_settings":              {
-            "input_type":       "mdpa",
-            "input_filename":   "line_body_all_stage_new_units_kPa_stage2"
+            "input_type":       "mdpa"
```

As seen in my previous comment, for clarity:
```suggestion
            "input_type":       "use_input_model_part"
```

---

Richard, thank you very much for the clarification. I changed mdpa onto use_input_model_part.